src/NHibernate/Linq/Visitors/NhPartialEvaluatingExpressionVisitor.cs

```diff
@@ -157,7 +157,9 @@ private Expression EvaluateSubtree(Expression subtree)
 
 		private bool ContainsVariable(Expression expression)
```

Comment:
@maca88 can this method be moved to `NhEvaluatableExpressionFilter`?

Reply:
Unfortunately not, it was placed here due to #2458, to evaluate expression like `string.Fromat("{0}", variable)`.